DebbeqGasha/DG001.xml

```diff
@@ -43,11 +43,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                                   <msItem xml:id="ms_i2">
                                       <locus from="51r" to="62v"/>
-                    <title type="complete" ref=""> </title>
+                    <title type="Incomplete" ref="LIT6510PrayerEstenfasa"> </title>
```

Comment:
If you have information on the incompleteness (is something at the beginning missing? at the end?), you could write this in a `<note>`